resources/assets/coffee/react/beatmapset-page/score.coffee

```diff
@@ -18,16 +18,18 @@
 {div, a} = React.DOM
 el = React.createElement
 
-class BeatmapsetPage.ScoreboardItem extends React.Component
+BeatmapsetPage.Score = React.createClass
+  mixins: [React.addons.PureRenderMixin]
+
   render: ->
-    elements = ['rank', 'flag', 'player', 'mods', 'icons', 'score', 'accuracy']
+    div className: 'beatmapset-score',
+      for elem in ['position', 'flag', 'player', 'mods', 'rank', 'score', 'accuracy', 'hits']
```

Comment:
so i assume that you want me to just type out each element manually without any loops?